mypyc/irbuild/ll_builder.py

```diff
@@ -1124,14 +1134,13 @@ def new_tuple(self, items: List[Value], line: int) -> Value:
         size = Integer(len(items), c_pyssize_t_rprimitive)  # type: Value
         return self.call_c(new_tuple_op, [size] + items, line)
 
-    def new_tuple_with_length(self, val: Value, line: int) -> Value:
-        """Generate a new empty tuple with length from a list or tuple
+    def new_tuple_with_length(self, length: Value, line: int) -> Value:
+        """Generate a new empty tuple with length
```

Comment:
Document here that the object must be initialized afterwards before it can be visible to user code.